internal/config/database.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 type database struct {
-	Dir                   string `mapstructure:"cache-dir"`
-	UpdateURL             string `mapstructure:"update-url"`
-	AutoUpdate            bool   `mapstructure:"auto-update"`
-	ValidateByHashOnStart bool   `mapstructure:"validate-by-hash-on-start"`
+	Dir                   string `yaml:"cache-dir" json:"cache-dir" mapstructure:"cache-dir"`
+	UpdateURL             string `yaml:"update-url" json:"update-url" mapstructure:"update-url"`
+	AutoUpdate            bool   `yaml:"auto-update" json:"auto-update" mapstructure:"auto-update"`
+	ValidateByHashOnStart bool   `yaml:"validate-by-hash-on-start" json:"validate-by-hash-on-start" mapstructure:"validate-by-hash-on-start"`
```

Comment:
Blocking: for JSON, we use `camelCase`

Reply:
Disregard, and I'll approve again: I see that we tend to use `snake-case` for our configuration JSON.

So given that, I'd suggest that as we think about important but breaking changes we want to make for 1.0, we consider using a consistent JSON convention between our inputs and outputs. 🙏

Reply:
that makes sense 💯